pam/internal/adapter/nativemodel.go

```diff
@@ -584,13 +584,13 @@ func (m nativeModel) handleFormChallenge(hasWait bool) tea.Cmd {
 		}
 	}
 
-	instructions := "Insert '%[1]s' to cancel the request and go back"
+	instructions := "Type '%[1]s' to cancel the request and go back"
 	if hasWait {
 		// Duplicating some contents here, as it will be better for translators once we've them
-		instructions = "Leave the input field empty to wait for other authentication method " +
+		instructions = "Leave the input field empty to wait for the alternative authentication method " +
 			"or insert '%[1]s' to go back"
```

Comment:
"insert" isn't the best choice, but since we already mention the input field here, I think "enter" is better than "type".

```suggestion
			"or enter '%[1]s' to go back"
```